dominion/dominion_ui/public/index.html

```diff
@@ -59,14 +59,23 @@
     <!-- test purposes -->
     <script type="text/javascript">
         var MAPIT = { url: "https://mapit.hurumap.org" };
+        var dominion_countries = {
+            'south-africa': {
+                name: 'South Africa'
+            },
+            'kenya': {
+                name: 'Kenya'
+            }
+        }
     </script>
   </head>
 
-  <body>
+  <body style="background-image: url(/static/img/background.png);">
```

Comment:
@karimkawambwa I am sure this works running dominion_ui as a standalone. If you run dominion hurumap-app, we don't have a background :open_mouth: 
Either we add this css for dominion (django app) templates  or have it set on the components
![screenshot from 2019-03-07 13-17-52](https://user-images.githubusercontent.com/7962097/53949902-14f39000-40dc-11e9-99dc-9977970166b1.png)

Reply:
Pushed the changes already.
I forgot the setup @kilemensi had does not import css from react app.

Reply:
Mhh... we don't _css_ in react @karimkawambwa, we _css in js_. This feels like the background should be applied to the header component _in react_ to me (header = nav + hero).